packages/react-debug-tools/src/__tests__/ReactHooksInspectionIntegration-test.js

```diff
@@ -833,6 +833,44 @@ describe('ReactHooksInspectionIntegration', () => {
     `);
   });
 
+  it('should inspect the value of the current provider in useContext reading the same context multiple times', async () => {
```

Comment:
In order to run this test against React >=17, you should include `// @reactVersion >= 17.0` at the top

Reply:
Opted for 16.8 since that's when hooks got introduced.